i2c-imx: allow autoloading on according to dt ids

[ not yet sent upstream ]

Allow udev to autoload the module when booting with device-tree

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
Index: linux/drivers/i2c/busses/i2c-imx.c
===================================================================
--- linux.orig/drivers/i2c/busses/i2c-imx.c	2013-04-29 02:36:01.000000000 +0200
+++ linux/drivers/i2c/busses/i2c-imx.c	2013-05-16 08:56:06.000000000 +0200
@@ -148,6 +148,7 @@ static const struct of_device_id i2c_imx
 	{ .compatible = "fsl,imx21-i2c", .data = &imx_i2c_devtype[IMX21_I2C], },
 	{ /* sentinel */ }
 };
+MODULE_DEVICE_TABLE(of, i2c_imx_dt_ids);
 
 static inline int is_imx1_i2c(struct imx_i2c_struct *i2c_imx)
 {
